src/routes/+error.svelte

```diff
@@ -9,13 +9,31 @@
 <section title={$page.error?.message}>
   <em>404</em>
   <h1>Frank can't find where you're going!</h1>
+  <a href="/">Return to Home</a>
```

Comment:
For this scenario, we have a prebuilt common button component. Feel free to update our common button with your fine-tuned styles, just use the button imported like so:

```ts
  import Button from '$lib/components/button/button.svelte';
  ```

Reply:
Added changes and found one issue, read comment below